its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -137,9 +137,7 @@ public static ScannerForMSBuild createEndStep(Path projectDir) {
   }
 
   public static ScannerForMSBuild createBeginStep(String projectName, Path projectDir) {
-    return createBeginStep(projectName, projectDir, "")
-      // FIXME: Remove the next line after this issue is resolved: https://sonarsource.slack.com/archives/CJ8L44WRF/p1704444673166759
-      .setProperty("sonar.plugins.downloadOnlyRequired", "false");
```

Comment:
I'd suggest to move this property just in the Empty project test, so that we keep the tested functionality (an empty project / folder with no CS file).

Otherwise, we're changing the scenario and that folder isn't empty anymore

where is that folder being used in tests?

Reply:
from discussion - it is used for two different purposes:
- to populate the plugin cache when starting orchestrator (so without adding the empty C# file, this wouldn't work anymore, possibly leading to race conditions again when all tests run in parallel)
- for the test about analysis warnings - which seems to only have reused that folder and it also needs the plugin to be downloaded in order to work